winui/Scheduler/Getting-Started.md

```diff
@@ -78,7 +78,7 @@ this.Schedule.ViewType = SchedulerViewType.Month;
 {% endhighlight %}
 {% endtabs %}
 
-![show-month-view-in-winui-scheduler](GettingStarted_Images/month-view-in-winui-scheduler.png)
+![Show month view in WinUI Scheduler](gettingstarted_images/winui-scheduler-show-month-view.png)
```

Comment:
Alt text is Fine. 
But existing image name is fine. revert your changes in image name alone for all scheduler change.